pkg/kubernetes/secrets/store.go

```diff
@@ -24,54 +24,56 @@ const (
 
 // Store implements the backing store for secrets as kubernetes secrets.
 type Store struct {
-	logger    hclog.Logger
-	config    config.Config
-	hostStore cnabsecrets.Store
-	clientSet *kubernetes.Clientset
+	*portercontext.Context
+	hostStore  cnabsecrets.Store
+	Secrets    map[string]map[string]string
+	namespace  string
+	kubeconfig string
+	clientSet  *kubernetes.Clientset
+	logger     hclog.Logger
 }
 
-func NewStore(cfg config.Config, l hclog.Logger) cnabsecrets.Store {
+func NewStore(c *portercontext.Context, cfg PluginConfig) *Store {
+	namespace := cfg.Namespace
 	s := &Store{
-		config:    cfg,
-		logger:    l,
-		hostStore: &host.SecretStore{},
+		Secrets:   make(map[string]map[string]string),
+		hostStore: &cnabhost.SecretStore{},
+		namespace: namespace,
+		logger:    cfg.Logger,
 	}
-
-	return portersecrets.NewSecretStore(s)
+	return s
 }
 
-func (s *Store) Connect() error {
+func (s *Store) connect() error {
 
 	if s.clientSet != nil {
 		return nil
 	}
-
-	clientSet, namespace, err := k8s.GetClientSet(s.config.Namespace, s.logger)
+	s.logger.Info(fmt.Sprintf("Store.connect: pre-clientset %s : %s", "namespace", s.namespace))
```

Comment:
What's the significance of pre/post clientset? Is it for just that the kubeconfig's current context had a different namespace set than the store? Or is there more to it?

Reply:
No more to it.  It's to help troubleshoot if current env context is different than config

Reply:
Let's change this and any others to debug then perhaps? Or do the messages not get captured by porter unless they are info+?